rimage/lazy_encoded_test.go

```diff
@@ -39,22 +39,23 @@ func TestLazyEncodedImage(t *testing.T) {
 	imgLazy = NewLazyEncodedImage([]byte{1, 2, 3}, utils.MimeTypePNG)
 
 	test.That(t, imgLazy.(*LazyEncodedImage).MIMEType(), test.ShouldEqual, utils.MimeTypePNG)
-	test.That(t, func() { imgLazy.Bounds() }, test.ShouldPanic)
-	test.That(t, func() { imgLazy.ColorModel() }, test.ShouldPanicWith, image.ErrFormat)
-	test.That(t, func() { NewColorFromColor(imgLazy.At(0, 0)) }, test.ShouldPanicWith, image.ErrFormat)
-	test.That(t, func() { NewColorFromColor(imgLazy.At(4, 4)) }, test.ShouldPanicWith, image.ErrFormat)
+	test.That(t, imgLazy.(*LazyEncodedImage).DecodeAll(), test.ShouldNotBeNil)
+	test.That(t, func() { _ = imgLazy.Bounds() }, test.ShouldPanic)
+	test.That(t, func() { _ = imgLazy.ColorModel() }, test.ShouldPanic)
+	test.That(t, func() { NewColorFromColor(imgLazy.At(0, 0)) }, test.ShouldPanic)
 
 	imgLazy = NewLazyEncodedImage([]byte{1, 2, 3}, "weeeee")
 
 	test.That(t, imgLazy.(*LazyEncodedImage).MIMEType(), test.ShouldEqual, "weeeee")
-	test.That(t, func() { imgLazy.Bounds() }, test.ShouldPanic)
-	test.That(t, func() { imgLazy.ColorModel() }, test.ShouldPanic)
+	test.That(t, imgLazy.(*LazyEncodedImage).DecodeAll(), test.ShouldNotBeNil)
```

Comment:
Specific error check please.

Reply:
Added one that checks for substring since to my knowledge there is no error type we can assert and check for.